cmdLine/src/main/scala/topgun/cmdline/FileParser.scala

```diff
@@ -12,30 +15,91 @@ class FileParser(file: File, cmdLine: JfrParseCommandLine, totals: Totals, confi
 
   import configuration.{includeStack, includeThread}
 
-
+  var classLoaderKey = 0
 
   def parse(): Unit = {
 
     println(s"*** File $file")
-    var allocationInNewTlab: Boolean = false
-    var allocationOutsideTlab: Boolean = false
-    var methodProfilingSample: Boolean = false
+    var allocationInNewTlab: Boolean = true
```

Comment:
Options would be good here to see what you have seen

Reply:
done